components/recurring-contributions/RecurringContributionsPopUp.js

```diff
@@ -115,7 +115,7 @@ const RecurringContributionsPopUp = ({ contribution, status, onCloseEdit, accoun
             </Flex>
             <GrayXCircle size={26} onClick={onCloseEdit} />
           </Flex>
-          {account.type !== 'COLLECTIVE' && (
+          {account.type !== 'COLLECTIVE' && Boolean(LoggedInUser?.isAdminOfCollective(account)) && (
```

Comment:
A small comment to explain this will help clarify

```suggestion
          {/** This popup is also used by root users, and we don't want them to touch the payment methods */}
          {account.type !== 'COLLECTIVE' && Boolean(LoggedInUser?.isAdminOfCollective(account)) && (
```

Reply:
Done. 👍🏽